src/deadline/cinema4d_adaptor/Cinema4DClient/cinema4d_handler.py

```diff
@@ -68,8 +81,11 @@ def start_render(self, data: dict) -> None:
             c4d.RENDERFLAGS_EXTERNAL | c4d.RENDERFLAGS_SHOWERRORS,
             prog=progress_callback,
         )
-        if result != c4d.RENDERRESULT_OK and result != c4d.RENDERRESULT_USERBREAK:
-            print("Error: RenderDocument: %s" % result)
+        result_description = _RENDERRESULT.get(result, None)
+        if result_description is None:
```

Comment:
Can we have a test for these?

Reply:
We could but there's zero testing skeleton for handler at the moment. It seems like a larger project than narrowly  testing the correct handling of RenderDocument() result value. Do you see a simple way to test this code path without dragging that extra complexity and line count for this PR? Thanks!

Reply:
Hmm, I was thinking of something similar to how it was done in [one of the PRs](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/pull/72/files). 

Mocking this whole function may not be straightforward, but if we were to abstract this into a separate function then mocking that function could be simpler. 

For example, 

```
def _process_render_result(result, None):
     if result_description is None:
            raise RuntimeError("Error: unhandled render result: %s" % result)
        if result != c4d.RENDERRESULT_OK:
            raise RuntimeError("Error: render result: %s" % result_description)
        else:
            print("Finished Rendering")
```

By extracting this logic, we could more easily mock and test _process_render_result. This approach would also allow us to test different RenderResults such as AssetMissing, SavingFailed, etc.

But even after using this approach, there will still be huge gaps in testing as the handler code does not have any testing right now. So I'm fine if we cannot add tests now. :)